scalding-macros/src/test/scala/com/twitter/scalding/macros/MacroOrderingProperties.scala

```diff
@@ -0,0 +1,540 @@
+    check[java.lang.Float]
+  }
+  test("Test out Long") { check[Long] }
+  test("Test out jl.Long") {
+    implicit val a = arbMap { b: Long => java.lang.Long.valueOf(b) }
+    check[java.lang.Long]
+  }
+  test("Test out Double") { check[Double] }
+  test("Test out jl.Double") {
+    implicit val a = arbMap { b: Double => java.lang.Double.valueOf(b) }
+    check[java.lang.Double]
+  }
+
+  test("Test out String") {
+    primitiveOrderedBufferSupplier[String]
+
+    check[String]
+    checkMany[String]
+  }
+
+  test("Test out ByteBuffer") {
+    primitiveOrderedBufferSupplier[ByteBuffer]
+    check[ByteBuffer]
+  }
+
+  test("Test out List[Float]") {
+    primitiveOrderedBufferSupplier[List[Float]]
+    check[List[Float]]
+  }
+  test("Test out Queue[Int]") {
+    implicit val isa = collectionArb[Queue, Int]
+    primitiveOrderedBufferSupplier[Queue[Int]]
+    check[Queue[Int]]
+  }
+  test("Test out IndexedSeq[Int]") {
+    implicit val isa = collectionArb[IndexedSeq, Int]
+    primitiveOrderedBufferSupplier[IndexedSeq[Int]]
+    check[IndexedSeq[Int]]
+  }
+  test("Test out HashSet[Int]") {
+    import scala.collection.immutable.HashSet
+    implicit val isa = collectionArb[HashSet, Int]
+    primitiveOrderedBufferSupplier[HashSet[Int]]
+    check[HashSet[Int]]
+  }
+  test("Test out ListSet[Int]") {
+    import scala.collection.immutable.ListSet
+    implicit val isa = collectionArb[ListSet, Int]
+    primitiveOrderedBufferSupplier[ListSet[Int]]
+    check[ListSet[Int]]
+  }
+
+  test("Test out List[String]") {
+    primitiveOrderedBufferSupplier[List[String]]
+    check[List[String]]
+  }
+
+  test("Test out List[List[String]]") {
+    val oBuf = primitiveOrderedBufferSupplier[List[List[String]]]
+    assert(oBuf.dynamicSize(List(List("sdf"))) === None)
+    check[List[List[String]]]
+  }
+
+  test("Test out List[Int]") {
+    primitiveOrderedBufferSupplier[List[Int]]
+    check[List[Int]]
+  }
+
+  test("Test out SetAlias") {
+    primitiveOrderedBufferSupplier[SetAlias]
+    check[SetAlias]
+  }
+
+  test("Container.InnerCaseClass") {
+    primitiveOrderedBufferSupplier[Container.InnerCaseClass]
+    check[Container.InnerCaseClass]
+  }
+
+  test("Test out Seq[Int]") {
+    primitiveOrderedBufferSupplier[Seq[Int]]
+    check[Seq[Int]]
+  }
+  test("Test out scala.collection.Seq[Int]") {
+    primitiveOrderedBufferSupplier[scala.collection.Seq[Int]]
+    check[scala.collection.Seq[Int]]
+  }
+
+  test("Test out Array[Byte]") {
+    primitiveOrderedBufferSupplier[Array[Byte]]
+    check[Array[Byte]]
+  }
+
+  test("Test out Vector[Int]") {
+    primitiveOrderedBufferSupplier[Vector[Int]]
+    check[Vector[Int]]
+  }
+
+  test("Test out Iterable[Int]") {
+    primitiveOrderedBufferSupplier[Iterable[Int]]
+    check[Iterable[Int]]
+  }
+
+  test("Test out Set[Int]") {
+    primitiveOrderedBufferSupplier[Set[Int]]
+    check[Set[Int]]
+  }
+
+  test("Test out Set[Double]") {
+    primitiveOrderedBufferSupplier[Set[Double]]
```

Comment:
Actually, I already have a listset in the tests, and it uses the same code path. I think we're safe here. Can we drop this issue?